core/collection-factory/mapdb3/pom.xml

```diff
@@ -27,7 +27,7 @@
 		<dependency>
 			<groupId>org.mapdb</groupId>
 			<artifactId>mapdb</artifactId>
-			<version>3.0.9</version>
+			<version>3.1.0</version>
```

Comment:
@JervenBolleman Do you think this update is ok? I see that neither 3.0.10 or 3.1.0 were out when you last changed the version here. So I'm more wondering if you think that this would break something, or if you think that the test coverage is good enough.

Reply:
I saw you commented already :)

Reply:
I think this is ok. Test coverage is good enough.